entrypoint.sh

```diff
@@ -34,29 +36,39 @@ set -e
 : ${GANESHA_OPTIONS:="-N NIV_EVENT -L STDOUT"}
 : ${GANESHA_EPOCH:=''} # For restarting
 
+if [ ! -z "${KV_CA_CERT}" ]; then
```

Comment:
it is harmless though half baked :) 
carrying it is fine but no entrypoint exposure for now.